src/components/ButtonBox/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Button from "../Button/index.jsx";
+import styles from "./styles/styles.module.scss";
+
+const ButtonBox = ({ title }) => {
```

Comment:
This component is a generic component and should receive these props. For example:
`onNextButtonClick`, `nextButtonText`, `nextButtonType`
`onPrevButtonClick`, `prevButtonText`, `prevButtonType`
Consider above props as a way of thinking and come up with a new structure.

Reply:
emmm now I got this and handled